src/main/java/Controller/musicEngineController.java

```diff
@@ -107,6 +111,18 @@ public void playRecommendation(playlistInputData data){
     }
 
     /**
+     * Skips the current song, if a playlist is playing.
```

Comment:
Wouldn't hurt to add a quick explanation why we only care if a playlist is playing (ex. for the TAs LOL).

Reply:
fixed